rfd/0026/README.md

```diff
@@ -7,7 +7,7 @@ state: draft
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**
 
-- [RFD 26 Network Shared Storage for SDC](#rfd-26-network-shared-storage-for-sdc)
+- [RFD 26 Network Shared Storage for Triton](#rfd-26-network-shared-storage-for-Triton)
```

Comment:
That "Triton" in anchors should be "triton" (lowercase). I *think* URL anchors are case-sensitive.